web/src/lib/types/types.ts

```diff
@@ -463,11 +463,8 @@ export type SidebarCardContent = {
   header?: string;
   notStartedHeader?: string;
   completedHeader?: string;
-  imgPath: string | null;
   ctaText?: string;
-  color: string;
-  headerBackgroundColor: string | null;
-  borderColor: string;
+  preBodySpanButtonText: string;
```

Comment:
preBodySpanButtonText should be optional

Reply:
done